clicache/integration-test2/CqOperationTest.cs

```diff
@@ -87,8 +87,25 @@ public MyCqListener()
 
         public virtual void OnEvent(CqEvent<TKey, TResult> ev)
         {
-            Debug.WriteLine("MyCqListener::OnEvent called");
-            MyOrder val = ev.getNewValue() as MyOrder;
+        }
+
+        public virtual void OnError(CqEvent<TKey, TResult> ev)
+        {
+            Debug.WriteLine("CqListener::OnError called");
```

Comment:
Tests shouldn't be writing things to console. Shouldn't there be some flag this call is setting so that you can assert it was called later?

Reply:
We'll clean that up